generate-ctrl-event.ps1

```diff
@@ -13,4 +13,4 @@ public static class Kernel32
 }
 "@
 
-[Kernel32]::GenerateConsoleCtrlEvent(0, 0)
+[Kernel32]::GenerateConsoleCtrlEvent($args[0], 0)
```

Comment:
Shouldn't you also obtain `dwProcessGroupId` from `$args`?

Reply:
Wow I swear I made that change.. thanks for taking a look! Fixed in 401a7bec62761c701ec2e2fc11c1271d49642c54